generated/nidaqmx/nidaqmx.proto

```diff
@@ -847,6 +865,52 @@ enum WriteBasicTEDSOptions {
   WRITE_BASIC_TEDS_OPTIONS_DO_NOT_WRITE = 12540;
 }
 
+enum ScaleInt32AttributeValues {
```

Comment:
Note: I think it would be helpful to include the enum name in these names. This will be more important for channel attributes where there are *a lot* of different options.

Reply:
Interesting, I wonder why that's not working for this one. I'll look at it closer.